README.md

```diff
@@ -31,15 +31,15 @@ repositories {
 }
 
 dependencies {
-    compile "io.projectreactor:reactor-core:3.5.7"
-    testCompile "io.projectreactor:reactor-test:3.5.7"
+    compile "io.projectreactor:reactor-core:3.6.0-SNAPSHOT"
```

Comment:
As the README.md will be displayed in the repo's GitHub entry point, I think we shouldn't advertise using the snapshot here.

Reply:
reverted